pyuvsim/uvsim.py

```diff
@@ -403,6 +404,28 @@ def run_uvdata_uvsim(input_uv, beam_list, beam_dict=None, catalog=None):
     if rank == 0:
         print("Calculations Complete.")
 
+    # If profiling is active, save meta data:
+    from .profiling import prof     # noqa
+    if hasattr(prof, 'meta_file'):  # pragma: nocover
```

Comment:
Let's add a comment to the lines you are leaving out of coverage because of the interaction between line_profiler and the coverage calculator to the effect that they are in fact covered by tests and that the # pragma: nocover can be removed when that bug is fixed.

Reply:
as above